app/controllers/playlist_items_controller.rb

```diff
@@ -62,6 +62,41 @@ def update
     render json: { message: "Item was not updated: #{error.message}" }, status: 500 and return
   end
 
+  # GET /playlists/1/items/2/markers
```

Comment:
It looks like we need to add authorization checks to these three actions and corresponding tests to ensure they are working as expected (security and functionality).